x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer.js

```diff
@@ -421,6 +428,29 @@ export class TimeSeriesExplorer extends React.Component {
       );
   };
 
+  /**
+   * Loads the full list of annotations for job without any aggs or time boundaries
+   * used to indicate existence of annotations that are beyond the selected time
+   * in the time series brush area
+   */
+  loadAnnotations = async (jobId) => {
+    ml.annotations
+      .getAnnotations({
+        jobIds: [jobId],
+        earliestMs: null,
+        latestMs: null,
+        maxAnnotations: ANNOTATIONS_TABLE_DEFAULT_QUERY_SIZE,
+      })
+      .then((resp) => {
+        if (!this.unmounted && resp?.success === true) {
+          this.setState({ annotationData: resp.annotations[jobId] ?? [] });
+        }
+      })
```

Comment:
your function is `async`, so you probably don't need `then`

Reply:
Updated here 3fb8c08